packages/vc-test-suite/httpService.ts

```diff
@@ -9,6 +9,13 @@ export const request = async (params: AxiosRequestConfig) => {
     headers: defaultHeaders,
   });
 
+  // Check if user defined headers is an object and contains only string values
+  if (params.headers) {
```

Comment:
Hi, should we move this check before we call `axios.create`? If params.headers already contains conflicting values (like another Content-Type), they will not merge properly. The axios.create() method will only use defaultHeaders unless manually merged.

Reply:
Good catch